source/agent/addons/quic/binding.gyp

```diff
@@ -44,10 +44,9 @@
     ],
```

Comment:
Remove 'conn_handler' in 'include_dirs' in binding.gyp

Reply:
Done.